samples/core/Saving/Saving/Concurrency/Sample.cs

```diff
@@ -25,8 +25,10 @@ public static void Run()
                 var person = context.People.Single(p => p.PersonId == 1);
                 person.PhoneNumber = "555-555-5555";
 
-                // Change the persons name in the database (will cause a concurrency conflict)
-                context.Database.ExecuteSqlCommand("UPDATE dbo.People SET FirstName = 'Jane' WHERE PersonId = 1");
+                // Change the persons name in the database - causes a concurrency conflict
+                // Requires NuGet package Microsoft.EntityFrameworkCore.Relational
```

Comment:
Is this a common practice? I am afraid it would make the sample code very verbose if we did it always.

Reply:
Some place we need to tell folks this is a requirement to run the code. It doesn't have to be here, I could hide it in a readme.md file. We use it extensively in the asp.net repo because it cuts down on questions on why an extension method is not found. Intellisense is no help. I had to go to the API page, look up the API, then hunt for the DLL. I had to do that twice - alot of friction. How many customers will know these packages are required?